mimic/model/cloudfeeds.py

```diff
@@ -0,0 +1,86 @@
+"""
+Emulate the behavior of Rackspace Cloud Feeds service.
+"""
+
+import attr
+
+
+@attr.s
+class CloudFeedsProduct(object):
+    """
+    Models a single CloudFeed product endpoint and its respective
+    functionality.
+    """
+    title = attr.ib(attr.validators.instance_of(str))
+    href = attr.ib(attr.validators.instance_of(str))
+
+    events = attr.ib(default=attr.Factory(list))
+
+    def post(self, event):
+        """
+        Post an event to the product queue.
+        """
+        self.events.append(event)
+
+
+@attr.s
+class CloudFeeds(object):
+    """
+    Models CloudFeeds support at the plugin-level.
+    """
+    _endpoints = attr.ib(default=attr.Factory(dict))
+
+    def get_product_endpoints(self):
+        """
+        Return a list of product endpoints registered with this class.
```

Comment:
Docstring says list, but it actually returns a copy of the existing endpoints dictionary - it looks like the tests call `render_product_endpoints_dict`  with the result of this value, so is it supposed to be a list?